--- Mapping Configuration
# Survey Data ##################################################################
# # Where the blender track mesh is zeroed
UTMdatum: # 2 layer nested dictionary
    E: 659300 
    N: 3607850
#
# # Locations of stripe lat/lon survey points
survey_stripe_locs: 
- '/home/gavlab/rgc0003rosws/fhwa2_viz/fhwa2_MOOS_to_ROS/survey/stripes/inner.txt'
- '/home/gavlab/rgc0003rosws/fhwa2_viz/fhwa2_MOOS_to_ROS/survey/stripes/middle.txt'
- '/home/gavlab/rgc0003rosws/fhwa2_viz/fhwa2_MOOS_to_ROS/survey/stripes/outer.txt'
#
# # Locations of lane center lat/lon survey points
survey_center_locs:
- '/home/gavlab/rgc0003rosws/fhwa2_viz/fhwa2_MOOS_to_ROS/survey/centers/inner.txt'
- '/home/gavlab/rgc0003rosws/fhwa2_viz/fhwa2_MOOS_to_ROS/survey/centers/outer.txt'
#
# # File location for mesh of track
# # the syntax denotes a location in the ROS package
track_mesh_resource: "package://fhwa2_MOOS_to_ROS/mesh/NCAT_pavement_plane.stl"
#
# # File location for mesh of lane markings
# # syntax denotes a location in the ROS package
marking_mesh_resource: "package://fhwa2_MOOS_to_ROS/mesh/NCAT_lane_markings.stl"
#
################################################################################